projects/observability/src/shared/components/gauge/gauge.component.ts

```diff
@@ -107,23 +107,20 @@ export class GaugeComponent implements OnChanges {
   }
 
   private buildRadius(boundingBox: ClientRect): number {
-    return Math.min(
-      boundingBox.height - GaugeComponent.GAUGE_AXIS_PADDING,
-      boundingBox.height / 2 + Math.min(boundingBox.height, boundingBox.width) / 2
-    );
+    return Math.min(boundingBox.height - GaugeComponent.GAUGE_AXIS_PADDING, boundingBox.width / 2);
   }
 
-  private buildOrigin(boundingBox: ClientRect, radius: number): Point {
+  private buildOrigin(boundingBox: ClientRect): Point {
     return {
       x: boundingBox.width / 2,
-      y: radius
+      y: boundingBox.height - GaugeComponent.GAUGE_AXIS_PADDING
```

Comment:
We likely want this to center for tall containers, rather than always sit at the bottom of the container

Reply:
That would create alignment issues imo. By keeping it at the bottom, we can assign appropriate dimension to dash container to keep it aligned. Noticed this while adding this gauge to an existing dash. But can sure revisit.